src/components/views/dialogs/ChatInviteDialog.js

```diff
@@ -27,6 +27,10 @@ var Modal = require('../../../Modal');
 
 const TRUNCATE_QUERY_LIST = 40;
 
+function escapeRegExp(str) {
+  return str.replace(/[\-\[\]\/\{\}\(\)\*\+\?\.\\\^\$\|]/g, "\\$&");
```

Comment:
wow that's horrific. might be worth crediting where it came from (plus the indenting is wrong).